lib/config/options/index.ts

```diff
@@ -944,7 +944,7 @@ const options: RenovateOptions[] = [
   {
     name: 'matchBaseBranches',
     description:
-      'List of strings containing exact matches (e.g. `["main"]`) and/or regex expressions (e.g. `["/^release\\/.*/"]`). Valid only within a `packageRules` object.',
+      'List of strings containing exact matches (e.g. `["main"]`) and/or regex expressions (e.g. `["/^release/.*/"]`). Valid only within a `packageRules` object.',
```

Comment:
Do we trim the leading and trailing slash before passing to regex function? those can maybe change behavior. 🤔

Reply:
Yes, we do